src/components/Pagination/tests/Pagination.test.tsx

```diff
@@ -139,6 +139,24 @@ describe('<Pagination />', () => {
 
       expect(spy).toHaveBeenCalledTimes(1);
     });
+
+    it('does not navigate the browser when hasNext or hasPrevious is false', () => {
+      const spy = jest.fn();
```

Comment:
We've started naming the spy according to what it's spying on, so in this case it would be called `anchorClickSpy`.